bigtable/integration_test.go

```diff
@@ -972,6 +982,9 @@ func TestIntegration_Read(t *testing.T) {
 			err := table.ReadRows(ctx, test.rr, func(r Row) bool {
 				for _, ris := range r {
 					for _, ri := range ris {
+						if got, want := ri.Labels, test.wantLabels; !reflect.DeepEqual(got, want) {
+							t.Fatalf("got %q\nwant %q", got, want)
```

Comment:
Is it possible to collect the labels and raise the failure outside the loop? (similar to `elt` on L996)